src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -14,55 +17,69 @@ import { PdfDataFetchResponse } from "../common";
 interface DriverOverviewData {
     driverName: string;
     date: Date;
-    tableData: DriverOverviewTableData[];
+    tableData: DriverOverviewTablesData;
     message: string;
 }
 
-const compareDriverOverviewTableData = (
-    first: DriverOverviewTableData,
-    second: DriverOverviewTableData
+const compareCollectionCentres = (
+    first: DriverOverviewRowData,
+    second: DriverOverviewRowData
 ): number => {
-    if (first.name < second.name) {
+    if (first.collectionCentre < second.collectionCentre) {
         return -1;
     }
-    if (first.name > second.name) {
+    if (first.collectionCentre > second.collectionCentre) {
         return 1;
     }
     return 0;
 };
 
-type ParcelsForDelivery = (Schema["parcels"] & {
+const comparePostcodes = (first: DriverOverviewRowData, second: DriverOverviewRowData): number => {
+    if (first.address.postcode && second.address.postcode) {
+        return first.address.postcode.localeCompare(second.address.postcode);
+    }
+    if (first.address.postcode) {
+        return 1;
+    }
+    if (second.address.postcode) {
+        return -1;
+    }
+    return 0;
+};
+
```

Comment:
essentially the idea here is implementing javascripts default sorting for the postcode - I thought this makes sense as it is what we are doing elsewhere

Reply:
Yep think that's a good idea - I think you can simplify this function to 
```
const comparePostcodes = (first: DriverOverviewRowData, second: DriverOverviewRowData): number => {
    if (first.address.postcode && second.address.postcode) {
        return first.address.postcode.localeCompare(second.address.postcode);
    }
    if (!first.address.postcode && !second.address.postcode) {
        return 0;
    }
    return first.address.postcode ? 1 : -1;
};
```